bin/faucet/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM rust:1.87-slim-bullseye AS builder
+
+WORKDIR /app
+COPY . .
+
+RUN cargo install --path bin/faucet --locked
+
+# Connect to the node using `host.docker.internal`. This assumes that the node is running on the 
+# same host machine.
+RUN miden-faucet init \
+    --faucet-account-path faucet.mac \
+    --config-path miden-faucet.toml \
+    --node-url http://host.docker.internal:57291
+
+RUN miden-faucet create-faucet-account \
+    --output-path faucet.mac \
+    --token-symbol POL \
+    --decimals 9 \
+    --max-supply 1000000000
```

Comment:
This essentially means the faucet values are hardcoded for everyone, correct?

How it configuration usually handled for docker?

And I guess should we update the token, decimals and supply to match our new official values as per #945 or is this fine?